### DISCUSSION TITLE
`Could not resolve` full debug logs

### DISCUSSION THREAD

**Discussion by vjpr** [oven-sh/bun#1487] [Ideas]
At the moment there is no info about why a resolution failed.

```
error: Could not resolve: "@live/node/src/index.js". Maybe you need to "bun install"?

import {runNode} from '@live/node/src/index.js'
                      ^
/Users/Vaughan/dev-mono/thirtyfive/repos/live/packages/pkg-helpers/live-node-bun/src/index.ts:2:23 35%
```

From this message, I don't know if its because its not handling `package.json#exports` or it's not in `node_modules`, or what.

It would be nice to have a flag to enable a full debug log of why a module could not be found, including the entire resolution process.

Webpack does this and its very useful.

**Comment by Jarred-Sumner** [comment]
There is code for this, we are missing the setting to enable it + test coverage. Bun's module resolver is mostly based on esbuild's and has many of the same logging features
